drivers/ipm/Kconfig

```diff
@@ -67,6 +67,24 @@ config IPM_MHU
 	help
 	  Driver for SSE 200 MHU (Message Handling Unit)
 
+config IPM_NRFX
+	bool "IPM NRF driver"
+	depends on IPM && HAS_HW_NRF_IPC
+	select NRFX_IPC
+	help
+	  Driver for Nordic nRF messaging unit, based
+	  on nRF IPC peripheral HW.
+
+config IPM_NRF_SINGLE_INSTANCE
+	bool "Single instance of IPM device"
+	default n
```

Comment:
Can skip `default n`, for consistency. `n` is the default default.

That makes the generated docs a bit nicer too.

Reply:
Agree - default n needs to be removed, @kapi-no

Reply:
done